src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java

```diff
@@ -88,6 +88,7 @@ void converterTest(final String columnType, final String format, final String in
                     String.format("format.%s",
                             columnType.equals("timestamp") || columnType.equals("datetime2") ? "datetime" : columnType),
                     format);
+        props.put("debug", "true");
```

Comment:
Needed?

Reply:
Yes. I want to make sure the debug is turning on in Testing, for better visibility.